openapi/spec3.json

```diff
@@ -11,7 +11,12 @@
   },
   "servers": [
     {
-      "url": "https://api.meraki.com/api/v1"
+      "url": "https://api.meraki.com/api/v1",
```

Comment:
api/v1 here should be {basePath}

Reply:
so my understanding is that this is the final generated json, and it doesn't use the variable names, just the variable values. am I misunderstanding that?

Reply:
Hmmm I think the final generated JSON will have this {}. Here's a link to the doc: https://swagger.io/specification/#:~:text=The%20following%20shows%20how%20variables%20can%20be%20used%20for%20a%20server%20configuration%3A

Just like how we use the Liquid template.

Reply:
done